python/taichi/lang/impl.py

```diff
@@ -36,6 +37,12 @@ def expr_init_local_tensor(shape, element_type, elements):
         get_runtime().get_current_src_info())
 
 
+@taichi_scope
+def make_local_matrix(shape, element_type, elements):
```

Comment:
I suggest renaming it to `make_matrix_expr`. The result should be wrapped in `expr.Expr`.
References: https://github.com/taichi-dev/taichi/blob/2ba0343c904fb405086485cd5b853ff606e7b949/python/taichi/lang/impl.py#L224-L228
https://github.com/taichi-dev/taichi/blob/2ba0343c904fb405086485cd5b853ff606e7b949/taichi/python/export_lang.cpp#L974-L975